themes/bootstrap/static/js/color-console-output.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+let html = document.querySelector('section').innerHTML;
+
+// Find the ANSI escape sequences for coloring the text
+// const ansiPatterns = [/\^\[\[\d*m/g, /\^\[\[\d*;\d*m/g, /\^\[\[\d*;\d*;\d*m/g];
+// for (let pattern of ansiPatterns) {
+//   console.log(new Set(html.match(pattern)));  // Used in ansiSequencesToSpans
+// }
+
+// Manually format the relevant patterns
+const ansiSequencesToSpans = {
+  // '\\^\\[\\[0m': '<span>',
+  '\\^\\[\\[4m': '<span style="text-decoration:underline;">',
+  '\\^\\[\\[37m': '<span>',
+  '\\^\\[\\[37;44m': '<span style="background-color:tomato">',
+  '\\^\\[\\[30;42m': '<span style="color:white; background-color:steelblue">',
+  '\\^\\[\\[30;45m': '<span style="color:white; background-color:darkseagreen">',
+  '\\^\\[\\[38;5;232m': '<span style="color:#c7254e">',
+  '\\^\\[\\[38;5;236m': '<span>',
+  '\\^\\[\\[38;5;240m': '<span>',
```

Comment:
No color here?

Reply:
This will be resolved by a general ANSI code parser (see below).